mne/viz/evoked.py

```diff
@@ -2110,7 +2110,8 @@ def plot_compare_evokeds(evokeds, picks=None, gfp=False, colors=None,
         upper_v, lower_v = -ymax_bound, ax.get_ylim()[-1]
     else:
         upper_v, lower_v = ax.get_ylim()[0], ymax_bound
-    ax.vlines(vlines, upper_v, lower_v, linestyles='--', colors='k',
+    if vlines:
+        ax.vlines(vlines, upper_v, lower_v, linestyles='--', colors='k',
               linewidth=1., zorder=1)
```

Comment:
PEP8

Reply:
I'm really out of practice, okay? :D

And sorry for pushing to the main repo ...

Reply:
(If the last line was confusing, I created a branch on the main repo, but have since deleted it.)